tests/search/update_manager_test.py

```diff
@@ -27,19 +27,21 @@ def test_remove_all_old_docs(self):
         self.assertEqual(None, self.test_wrapper.get_metadata("delete"))
         self.assertEqual(None, self.test_wrapper.get_metadata("old"))
         self.assertEqual(None, self.test_wrapper.get_metadata("other"))
-        self.assertEqual(utils.get_meta_elastic("test_key"),
-                self.test_wrapper.get_metadata("test_key").to_dict()["property_list"])
 
     def test_remove_old_docs_per_bucket(self):
         key_list = ["test_key"]
         deleted = self.update_manager.remove_unlisted_documents_per_bucket(key_list, "test")
         self.assertEqual(2, deleted)
         self.assertEqual(None, self.test_wrapper.get_metadata("delete"))
         self.assertEqual(None, self.test_wrapper.get_metadata("old"))
-        self.assertEqual(utils.get_meta_elastic("test_key"),
-                self.test_wrapper.get_metadata("test_key").to_dict()["property_list"])
-        self.assertEqual(utils.get_meta_elastic("other", "/other/artifact/other"),
-                self.test_wrapper.get_metadata("other").to_dict()["property_list"])
+
+    def test_remove_documents_wildcard(self):
+        val_list = [
+            "/test/artifact/*"
```

Comment:
Instead of using one that already exists can you make up a fake one to delete?  Otherwise if we add another one to test in base, this test will fail.